pkg/inventory/db.go

```diff
@@ -272,8 +272,11 @@ func (db *DB) netdevToDRAdev(ifName string) (*resourceapi.Device, error) {
 
 	mac := link.Attrs().HardwareAddr.String()
 	// this is bounded and small number O(N) is ok
```

Comment:
Let's clean remove this comment fromhere, don't think it's relevant in this specific part of the code (and rather, it's now part of the internal implementation of getProviderAttributes)